avaframe/tests/test_com2AB.py

```diff
@@ -113,10 +113,10 @@ def test_com2ABMain(capfd):
     '''Simple test for function com2ABMain'''
     # load and prepare Inputs
     listNames = ['avaHockeySmall', 'avaHockeyChannel', 'avaBowl']
-    dirname = os.path.dirname(__file__)
+    dirname = pathlib.Path(__file__).parents[0]
     for name in listNames:
-        avalancheDir = os.path.join(dirname, '..', 'data', name)
-        saveOutPathRef = os.path.join(dirname, '..', '..', 'benchmarks', name + 'ABPytest')
+        avalancheDir = pathlib.Path(dirname, '..', 'data', name)
+        saveOutPathRef = pathlib.Path(dirname, '..', '..', 'benchmarks', name + 'ABPytest')
```

Comment:
dirName is already a pathlib object so you could use dirname / '..' / 'data' /name